R/ggMarginalHelpers.R

```diff
@@ -245,4 +245,10 @@ addRightMargPlot <- function(ggMargGrob, right, size) {
                                 b = panelPos[["b"]], r = ncol(gt), l = ncol(gt),
                                 z = Inf, clip = "on", name = "rightMargPlot")
   gt
+}
+
+getTitleGrob <- function(p) {
+  grobs <- ggplot2::ggplotGrob(p)$grobs
+  gind <- sapply(grobs, function(x) grepl(pattern = "plot.title", x$name))
```

Comment:
It should be, given that `getTitleGrob` is only called if we have already decieded there is a title:
```
  hasTitle <- (!is.null(pb$plot$labels$title))
  if (hasTitle) {
    title <- getTitleGrob(p = p)
    p$labels$title <- NULL
  }
```
That being said, the internal's of ggplot2 could change. We can include a unit test as part of the PR that will close #43, to make sure we are making the correct assumption about ggplot2 internals. However, I don't think it makes sense to add an assertion inside `getTitleGrob`, as we don't have them in other helper functions to `ggMarginal`.